include/dlaf/factorization/qr/t_factor_impl.h

```diff
@@ -248,31 +227,16 @@ struct Helpers<Backend::GPU, Device::GPU, T> {
 }
 
 template <Backend backend, Device device, class T>
-void QR_Tfactor<backend, device, T>::call(const SizeType k, Matrix<const T, device>& v,
-                                          const matrix::SubPanelView& panel_view,
+void QR_Tfactor<backend, device, T>::call(matrix::Panel<Coord::Col, T, device>& hh_panel,
                                           pika::shared_future<common::internal::vector<T>> taus,
                                           pika::future<matrix::Tile<T, device>> t) {
   using Helpers = tfactor_l::Helpers<backend, device, T>;
   // Fast return in case of no reflectors
-  if (k == 0)
+  if (hh_panel.getWidth() == 0)
     return;
 
-  const auto v_start = panel_view.offsetElement();
-  const auto v_start_tile = panel_view.offset();
+  const auto v_start = hh_panel.offsetElement();
 
-  if (device == Device::GPU && v_start.row() % v.blockSize().rows() != 0) {
-    DLAF_UNIMPLEMENTED(v_start, v.blockSize().rows());
-  }
-
-  const auto panel_width = panel_view.cols();
-
-  t = splitTile(t, {{0, 0}, {k, k}});
-
-  DLAF_ASSERT(k <= panel_width, k, panel_width);
-
-  const GlobalTileIndex v_end{v.nrTiles().rows(), std::min(v.nrTiles().cols(), v_start_tile.col() + 1)};
-
-  // TODO S/R
```

Comment:
Just a note in case you removed this TODO line inadvertently.

Reply:
Sender and receivers are there...